packages/apps/src/directory/ListingCard.tsx

```diff
@@ -100,7 +101,11 @@ const ListingCard: React.FC<ListingCardProps> = (props) => {
             >
               {brand?.title}
             </Link>
-            <Link href={listingHref} variant="h6">
+            <Link
+              href={!itemOnClick && listingHref}
+              onClick={() => itemOnClick(item)}
```

Comment:
exposed event handler props should start with 'on' prefix. So onItemClick. Also because the word item is too generic can mean anything but in this case refers to the title, would do onTitleClick. 

But finally, my recommendation would be to expose titleProps and then manage the onClick outside so that we get more flexibility without limiting the props to narrow use cases.